src/cs2_demo_downloader/match_scraper.py

```diff
@@ -1,36 +1,47 @@
 import logging
 import pickle
-import re
+import click
 from pathlib import Path
+from datetime import datetime, timezone, timedelta
 
 import steam.webauth as steam_auth
 from bs4 import BeautifulSoup
 from playwright.sync_api import sync_playwright
 
-from downloader import url_downloader
+from cs2_demo_downloader.downloader import url_downloader
```

Comment:
`from .downloader import url_downloader` is the better way to get this done - importing the module from within itself would mean it is its own dependency.